contrib/Dockerfile.test

```diff
@@ -12,6 +12,7 @@
 
 ARG GOLANG_VERSION=1.16.6
 
+# TODO: use ghcr.io
 FROM golang:${GOLANG_VERSION} AS golang-base
```

Comment:
@mxpv  Can we host Go as well?

Reply:
Done: ghcr.io/containerd/golang:1.16.6

Reply:
I don't think this Dockerfile is used in CI, so not sure it needs switching